tests/features/import-explorer.feature

```diff
@@ -0,0 +1,136 @@
+Feature: Importer Explorer - View and configure importers on importers details page
+  Background:
+      Given: The user navigates to TPA URL and logs into the TPA application
+
+Scenario: The user navigates to Importers Explore page
+  Given the user navigates to TPA URL page successfully
+  When the user selects the importers menu option
+  Then application navigates to Importers page
+  And the column headers Name, Type, Description, Source, Period and Status should be visible
+  And And Call to Action button for each importer should be visible
+
+Scenario: The user can check pagination controls on importers page
+  Given the user navigates to TPA URL page successfully
+  When the user selects the importers page from left navigation menu option
+  Then the importers page will be displayed with pagination controls with correct values
+  And it will be possible to move between pages accordingly
```

Comment:
Please change "will be" to "should be" on all "Then" keywords.

Reply:
Done